components-api/src/main/scala/pl/touk/nussknacker/engine/api/typed/CanBeSubclassDeterminer.scala

```diff
@@ -206,9 +212,19 @@ trait CanBeSubclassDeterminer {
     condNel(TypeConversionHandler.canBeConvertedTo(givenType, superclassCandidate), (), errMsgPrefix)
   }
 
-  // we use explicit autoboxing = true flag, as ClassUtils in commons-lang3:3.3 (used in Flink) cannot handle JDK 11...
-  private def isAssignable(from: Class[_], to: Class[_]): ValidatedNel[String, Unit] =
-    condNel(ClassUtils.isAssignable(from, to, true), (), s"$to is not assignable from $from")
 }
 
-object CanBeSubclassDeterminer extends CanBeSubclassDeterminer
+object CanBeSubclassDeterminer extends CanBeSubclassDeterminer {
+
+  // we use explicit autoboxing = true flag, as ClassUtils in commons-lang3:3.3 (used in Flink) cannot handle JDK 11...
+  def isAssignable(from: Class[_], to: Class[_]): Boolean = {
```

Comment:
Please add a test showing when lang3 solution works wrong

Reply:
Added in CanBeSubclassDeterminerSpec